src/module.ts

```diff
@@ -179,18 +179,20 @@ export default defineNuxtModule<ModuleOptions>({
 
     // Include CSS file in project css
     let resolvedCss: string
+    const { resolve: relativeResolve } = createResolver(import.meta.url)
+
     if (typeof cssPath === 'string') {
       if (existsSync(cssPath)) {
         logger.info(`Using Tailwind CSS from ~/${relative(nuxt.options.srcDir, cssPath)}`)
         resolvedCss = cssPath
       } else {
         logger.info('Using default Tailwind CSS file from runtime/tailwind.css')
         // @ts-ignore
-        resolvedCss = createResolver(import.meta.url).resolve('runtime/tailwind.css')
+        resolvedCss = relativeResolve('runtime/tailwind.css')
       }
     } else {
       logger.info('No Tailwind CSS file found. Skipping...')
-      resolvedCss = createResolver(import.meta.url).resolve('runtime/empty.css')
+      resolvedCss = relativeResolve('runtime/empty.css')
```

Comment:
Would `addTemplate({ name: 'empty.css', write: true, contents: () => '' }).dst` work?

Reply:
I prefer to keep our `runtime` for simplicity :)